tools/integration_tests/implicitdir/implicitdir_test.go

```diff
@@ -160,26 +188,14 @@ func TestMain(m *testing.M) {
 		os.Exit(1)
 	}
 
-	if err := mountGcsfuse(); err != nil {
-		log.Printf("mountGcsfuse: %v\n", err)
-		os.Exit(1)
-	}
-
-	log.Printf("Test log: %s\n", logFile)
-
-	// Creating a temporary directory to store files
-	// to be used for testing.
-	var err error
-	tmpDir, err = os.MkdirTemp(mntDir, "tmpDir")
+	flags := []string{"--experimental-enable-storage-client-library=true",
+		"--experimental-enable-storage-client-library=false",
+		"--implicit-dirs=true",
+		"--implicit-dirs=false"}
+	successCode, err := executeTest(flags, m)
 	if err != nil {
-		logAndExit(fmt.Sprintf("Mkdir at %q: %v", mntDir, err))
+		return
```

Comment:
in the earlier code when anything fails, we are doing os.exit(1), here you are simply calling return. What is the difference?

Reply:
ohh my bad, I though TestMain will return an error if any other error found. 
os.Exit(1) will terminate the program if there is any error.
Fixed it